### PR TITLE
Fix setupLocalDevelopment task regenerates demo storefront with error #8

### PR DESCRIPTION
**Pull request by guiliguili**
Fix [setupLocalDevelopment task regenerates demo storefront with error #8](https://github.com/sap-commerce-tools/ccv2-project-template/issues/8)

### CODE REVIEW

**Comment by guiliguili** [comment]
This pull request also fixes #11

File: core-customize/build.gradle.kts [hunk 1]
```diff
@@ -104,6 +92,57 @@ tasks.register<Exec>("symlinkSolrConfigForLocalDevelopment") {
...
+    dependsOn("bootstrapPlatform", "createDefaultConfig")
+
+    args("extgen")
+    antProperty("input.template", "yacceleratorordermanagement")
+    antProperty("input.name", "demoshopordermanagement")
+    antProperty("input.package", "com.demo.shop.ordermanagement")
+}
+
+// ant extgen -Dinput.template=yocc -Dinput.name=demoshopocc -Dinput.package=com.demo.shop.occ
+tasks.register<HybrisAntTask>("generateDemoOcc") {
```
Comment (mpern) [inline core-customize/build.gradle.kts:116]: Why do you generate two occ extensions? Just to show how it works?

Comment (guiliguili) [reply core-customize/build.gradle.kts:116]: As discussed Custom OCC extensions are not strictly needed if you use the CX recipes for demo purposes... But in a Spartacus project that’s most likely something that will be required C.f. https://help.sap.com/viewer/9d346683b0084da2938be8a285c0c27a/2005/en-US/d46d19516961438f8939718e87ed787b.html

There’s also the the demoshopordermanagement extension which replaces the demoshopfulfilmentprocess extension in the `cx` recipe

**Comment by mpern** [comment]
All changes are included in the latest release.

**Pull request closed without merging**